src/middlewared/middlewared/plugins/directoryservices_/cache.py

```diff
@@ -144,7 +142,6 @@ async def query(self, objtype, filters, options):
 
         `objtype`: 'USERS' or 'GROUPS'
         """
-        res = []
```

Comment:
Unused `res` is removed - fine.  However, it is referenced in a comment a few lines down.  Does this need to be updated?